core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1118,6 +1126,23 @@ public boolean isForNewerHudson() {
             }
         }
 
+        /**
+         * Returns true iff the plugin declares a minimum Java version and it's newer than what the Jenkins master is running on.
+         *
+         * @return true iff the plugin declares a minimum Java version and it's newer than what the Jenkins master is running on.
```

Comment:
Avoid duplicating text between summary and `@return` tag. You can omit one or the other, or use the `@return` tag to delve into details of format, while the summary gives the overall purpose.

Reply:
done